src/wp-includes/rest-api/endpoints/class-wp-rest-template-revisions-controller.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * REST API: WP_REST_Template_Revisions_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 6.4.0
+ */
+
+/**
+ * Core class used to access revisions via the REST API.
```

Comment:
As noted in autosaves controller, this should specify that this controller is specific to "template" types. See also the [`register_routes()` function description](https://github.com/WordPress/wordpress-develop/pull/3533/files?w=1#diff-02f46e74e0e933f227d20de793a7e189308144a4dbdad5dac0036657488705faR66) in both new controllers; in this one in particular, "based on post types supporting revisions." feels overly general.